architecture-building-systems/CityEnergyAnalyst
# I998 update shp for zug

This PR resolves issue #998 
There are two parts of changes:
1. In `reference-case-zug`, the `DC` and `DH` are updated with new .shp files. These files are generated from the `network_layout` scripts in cea.
2. In the cea, the scripts within the `network_layout` folder are updated to read `cea.config`

To test:
1. Update cea-reference-case from master, and run `thermal_network_matrix` for both `DC` and `DH` in `reference-case-zug` 
2. In cea, run `substations_location.py`, `connectivity_potential.py`, and `minimum_spanning_tree.py` (follow the Readme to check outputs)
